metricbeat/module/kubelet/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
package comment should be of the form "Package kubelet ..."